Upgrade-Guide/op_version.md

```diff
@@ -0,0 +1,22 @@
+
+###op-version
+op-version is the operating version of the Gluster which is running.
+
+op-version was introduced to ensure gluster bits running with different version of bits do not end up in a problem and backward compatibility issues can be tackled.
+
+After Gluster upgrade, it is advisable to have op-version updated.
+
+###Updating op-version
+
+Current op-version can be queried as below:
+
+    [root@~]#gluster volume get  <volume name> cluster.op-version
+
+op-version can be updated as below.
+For example, after upgrading to glusterfs-3.7.1, set op-version as:
+
+    [root@~]#gluster volume set all cluster.op-version 30701
```

Comment:
this is why I mention that document is still incomplete in one of my earlier comment. Currently there is no way for an user to figure out what should be the op-version until and unless a metric is documented or the RFE I had mentioned in implemented.